libs/ui/src/lib/facets/facets.model.ts

```diff
@@ -0,0 +1,18 @@
+export interface HasPath {
```

Comment:
I don't understand the name of the itnerface

Reply:
I wanted to express interface composition, with both models extending over the capability to have a path. If it's not clear or not useful, I can revert to both model having a "path" property.

Reply:
Oh ok got it, ok we can keep it.